config/routes.rb

```diff
@@ -11,13 +11,14 @@
 
   namespace :api do
     namespace :v1 do
-      resources :donuts, only: [:index, :show]
-      resources :reviews, only: [:index, :show]
-      resources :shops, only: [:index, :show]
+      resources :shops, only: [:index, :show] do
+        resources :donuts, only: [:index, :show] do
+          resources :reviews, only: [:index, :show]
+      end
     end
   end
-
-  get "*path", to: "donuts#index"
+end
+  get "*path", to: "shops#index"
 
```

Comment:
unfortunately, you should not have a catch all path e.g. *path. Instead, you should have explicit routing e.g. 
```
get '/shops', to: 'shops#show'
```

Reply:
done! thanks for the hint :)